R/boxes.R

```diff
@@ -278,25 +280,24 @@ box <- function(..., title = NULL, footer = NULL, status = NULL,
     titleTag <- h3(class = "box-title", title)
   }
 
+  boxTools <- NULL
   collapseTag <- NULL
-  if (collapsible) {
-    buttonStatus <- status %OR% "default"
 
+  if (collapsible) {
     collapseIcon <- if (collapsed) "plus" else "minus"
 
-    collapseTag <- div(class = "box-tools pull-right",
-      tags$button(class = paste0("btn btn-box-tool"),
-        `data-widget` = "collapse",
-        shiny::icon(collapseIcon)
-      )
-    )
+    collapseTag <- tags$button(class = "btn btn-box-tool",
+                               `data-widget` = "collapse",
+                               shiny::icon(collapseIcon))
   }
 
+  boxTools <- div(class = "box-tools pull-right", collapseTag, boxMenu)
+
   headerTag <- NULL
-  if (!is.null(titleTag) || !is.null(collapseTag)) {
+  if (!is.null(titleTag) || !is.null(boxTools)) {
```

Comment:
Wouldn't `boxTools` always be non-NULL? I think the `boxTools` div should be created only if it's needed.